features/step_definitions/user_management_steps.rb

```diff
@@ -46,7 +46,11 @@
 end
 
 Given(/^I am not an admin user$/) do
-  @current_user = Comfy::Cms::User.create!(admin: false, email: 'test@tester.com', password: 'password')
+  @current_user = Comfy::Cms::User.create!(
```

Comment:
Hi @KateWilkinson :), this can be easily replaced with a FactoryGirl for 'user'. 

See here: https://github.com/moneyadviceservice/cms/blob/master/spec/factories/users.rb